### ISSUE TITLE
Raise chances of tator borg

### ISSUE THREAD

**Issue by ParagonPrime** [yogstation13/yogstation-classic#1066]
Tator cyborg is a really, really, REALLY rare thing that only ever happens (rarely) on lowpop.

Change it. Tator borg is incredibly fun and interesting and need a higher chance of occurring.

**Comment by FluffySurvivor** [comment]
what 

traitor borg actually exists?

**Comment by ParagonPrime** [comment]
^exactly

yes, it exists, but it's superduper rare even when the rare conditions are met

**Comment by Time-Green** [comment]
It actually exists???
I have never ever seen it in my entire life.

**Comment by ShadowDeath6** [comment]
If this happens it's a bug
I've never seen this happen and I've never heard of it happening. 
The only thing I could think of is if an AI was supposed to be traitor but there was no AI so it made borg a traitor, but borgs should be blacklisted from traitor

**Comment by AsV9** [comment]
Borg traitor is a thing last I checked but it's horrendously rare.

**Comment by ParagonPrime** [comment]
tl;dr, tator borg is much like (former) tator AI- two assassination objectives and an 'escape' or 'survive' objective (can't remember which). The boon is, you can emag yourself at any time, and _are immune to being blown by the robotics control console_

but yeah it's horrifically rare because it _requires that there isn't an AI at roundstart_ and is, generally, apparently intentionally, made to be a fucking nightmarishly low chance of occurring

**Comment by X-TheDark** [comment]
I mean, an all-access (without any traces aka emag) fast borg that is spaceworthy and can have some pretty impressive stuff (laser rifle/energy sword/lube) sounds like a powergamers dream.

**Comment by Bekons** [comment]
Traitor borg is an unique antag type. He appears slaved to AI yet he's not. He has the ability to hack himself, which will make AI see him as unslaved, unlock himself (works only once) and have any emagged moduel items appear once he hacks himself.

Once i was one myself and people thought i was bugged.

**Comment by ParagonPrime** [comment]
yes

it's so fucking rare people assume it's a fucking bug

yes, it may be a powergamer's dream- but isn't tator AI, too? shortly after the murderbone rule change, tator AI just became "how quickly can i kill the WHOLE STATION 4noraisin", and yet nobody ever complains about it

**Comment by ShadowDeath6** [comment]
yeah but AI is stuck in one place and is extremely easy to kill unless it floods plasma and that got nerfed by lag killer

**Comment by FluffySurvivor** [comment]
> powergaming antag

**Comment by X-TheDark** [comment]
What I meant by that is that people who try to be as robust as possible whenever they are traitors would certainly not say "No" to All-Access + whatever-emagged-module-have-you + spaceworthy + etc combination entity that can shock/bolt doors to prevent pursuit.

**Comment by ShadowDeath6** [comment]
How to win traitor borg
Kill AI immediately
Shock every door
Kill all of science
Space ion rifle

**Comment by ghost** [comment]
get killed by a lucky flash from a bored librarian's pointer

**Comment by ParagonPrime** [comment]
^

Borgs in general are simultaneously OP and horrifically easy to kill if you know what you're doing.

**Comment by FluffySurvivor** [comment]
who plays librarian

**Comment by Robertruler77** [comment]
Me when I have to choose bewtween librarian or assistant.
